store/splus.js

```diff
@@ -19,39 +19,43 @@ function defaultWeek() {
 }
 
 /**
- * Return all lectures for the given timetable and week.
+ * Return all events for the given timetable and week.
  */
-async function loadLectures(timetable, week, $get) {
-  const ids = Array.isArray(timetable.id) ? timetable.id : [timetable.id];
+export async function loadEvents(timetable, week, $get) {
```

Comment:
warum übergibst du hier die Funktion z.B. axios?

Reply:
Mir fällt nichts besseres ein, wie ich an das `$get` sonst komme…

Reply:
kannst du nicht einfach axios so importieren und benutzen?

Reply:
Das Nuxt-Plugin setzt API-Port, API-Pfad usw. und das möchte ich nicht duplizieren.
Man könnte mit `loadEvents.apply(this, [timetable, week])` das this auf das vom Aufrufer setzen, das ist aber auch komisch 🙁